src/Resources/config/grids/admin.yaml

```diff
@@ -0,0 +1,4 @@
+imports:
+    - { resource: "@BitBagSyliusBannerPlugin/Resources/config/grids/admin/ad.yaml" }
```

Comment:
hmm, what do you think about loading all files into directories? (instead of each file separately)